src/modules/orders/state/orderBookAtom.ts

```diff
@@ -0,0 +1,5 @@
+import { atom } from 'jotai'
+
+import { EnrichedOrder } from '@cowprotocol/cow-sdk'
+
+export const orderBookAtom = atom<EnrichedOrder[]>([])
```

Comment:
This name is confusing me. Where does it come from?

I would imagine orderbook to be all orders in the protocol/pair. Is that what's stored here?

Reply:
This atom stores data from order-book API: https://api.cow.fi/docs/#/
Because of it, it has the name.
I'm happy to give any other name, but can't imagine any better :(

Reply:
Renamed to `apiOrdersAtom`, thank you!